CMakeLists.txt

```diff
@@ -109,38 +109,35 @@ endif("${isSystemDir}" STREQUAL "-1")
 set(GISBASE ${CMAKE_BINARY_DIR}/gisbase)
 file(TO_NATIVE_PATH "${CMAKE_SOURCE_DIR}" MODULE_TOPDIR)
 file(TO_NATIVE_PATH "${GISBASE}" GISBASE_NATIVE)
-file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/bin" BIN_DIR)
-file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/lib" LIB_DIR)
+file(TO_NATIVE_PATH "${GISBASE}/bin" BIN_DIR)
```

Comment:
Does this mean that simply deleting the build directory will not be enough to clean everything? Is it already doing part of the job of installing during the build step as a side effect?
If so, the changes in this commit seem like a bad idea to me.

Reply:
That should be enough because `GISBASE` is in `CMAKE_BINARY_DIR`.